freqtrade/__init__.py

```diff
@@ -1,27 +1,14 @@
 """ Freqtrade bot """
-__version__ = 'develop'
-
-if __version__ == 'develop':
+__version__ = 'dev'
```

Comment:
in case you're wondering - it's this line which makes me doubt - and have a look at basically everything you didn't change :laughing:

Reply:
Sorry, I don't understand? What to you doubt? What specific about this line?

Also everything I didn't change is quite a lot 😋

Reply:
My problem is the change from develop to dev.
I'm not sure if we use that someplace.

i'll probably revert that line for now - as it provides no benefit over the current method.